docs/develop/contributing.md

```diff
@@ -60,22 +64,18 @@ You can automatically create a ready-to-go `aaq-core` conda environment with:
 
         git checkout -b feature
 
-2. Make your changes and then run `pre-commits` if you have not set it up to run automatically.
-
-        pre-commit run --all  # 👈 to run it manually
```

Comment:
This should not be manual! I ran this, saw that we've accumulated lots of uncleaned files (extra whitespace, no new line, etc.). Not a huge deal but keeps commits & PRs clean